draft-kazuho-httpbis-priority.md

```diff
@@ -110,14 +110,15 @@ Unknown parameters MUST be ignored.
 ## urgency
 
 The `urgency` parameter takes one of the following sh-tokens as the value, that
-indicates how an HTTP response affects the processing of other responses:
+indicates how an HTTP response affects the usability of other responses:
```

Comment:
```suggestion
indicates how an HTTP response affects the usage of other responses:
```

usability seems like the wrong word here (it is human-focused). For example, a blocking CSS resource doesn't make the final rendered document less usable (unless it is really bad CSS ;) )

Reply:
Thanks you for the suggestion. While I think that we want to be human-focused for the browser use-case, I agree that it might be too nuanced.